httpload/pkg/loader/config.go

```diff
@@ -9,10 +9,11 @@ import (
 
 // Config manages the configuration parameters passed on the command line
 type Config struct {
-	nWorkers    int
-	nRequests   int
-	appDuration time.Duration
-	url         string
+	nWorkers                     int
+	nRequests                    int
+	appDuration                  time.Duration
+	maxQueriesPerSecondPerWorker int
```

Comment:
nit/style: better to use a shorter name to improve readability (ex: `qps` and add a comment)

Reply:
If I must add a comment the readability is decreasing, not improving. What am I missing here?

Reply:
If for example you write 
```
maxQPS int // maxQPS is the maximum number of queries per second
```
Then the reader gets what the value is by looking at the comment and the readability of the code is improved IMHO since you know do not need to parse a loong string to understand that this variable is the maximum number of queries per second, you can focus on the rest of the code, that is, how is this variable used and the logic around it.

I refer you to https://github.com/golang/go/wiki/CodeReviewComments#variable-names that complete and probably explain better the point I'm trying to make